pkgs/applications/networking/mailreaders/thunderbird/sequoia.nix

```diff
@@ -0,0 +1,43 @@
+{ lib, thunderbird, sequoia-octopus-librnp }:
+let
+  originalPackage = thunderbird;
+
+  # We use `overrideAttrs` instead of defining a new `mkDerivation` to keep
+  # the original package's `output`, `passthru`, and so on.
+  thunderbird-sequoia = originalPackage.overrideAttrs (old: rec {
+    pname = "thunderbird-sequoia";
+    version = "${old.version}+${sequoia-octopus-librnp.version}";
+
+    meta = with lib; {
+      description = "Thunderbird with the Sequoia OpenPGP backend";
+      homepage = "https://gitlab.com/sequoia-pgp/sequoia-octopus-librnp";
+      license = old.meta.license;
+      maintainers = with maintainers; [ puzzlewolf ];
+    };
+
+    buildCommand = ''
+      set -euo pipefail
```

Comment:
I think these settings are already present during any build, so it's better to remove this line.

Reply:
ok, done